### DISCUSSION TITLE
Hola a todos,recien estoy empezando mis tareas muuuy atrazadas

### DISCUSSION THREAD

**Discussion by gladys5** [Launch-X-Latam/MisionFrontEnd#466] [Team Posts]
Pero me gustaria feedback de su parte ,y si alguien quisiera hacer grupo para practicar me encantaria ponernos de acuerdo    [] https://github.com/gladys5/lauch-X-practicas)

**Comment by gumodi35** [comment]
Hola gladys5! Creo que tu practica cumple con los requerimientos, si necesitas alguna ayuda para realizar tus practicas comentame y con gusto puedo ayudarte, creo que habra tiempo de para que entregues las practicas asi que no te estreses, continua a tu ritmo. Saludos!

**Comment by gladys5** [comment]
Gracias

**Comment by gladys5** [comment]
Hola, Gracias por responder ,la verdad que si me veo todavia muy lejos de los objetivos del curso y espero resivir tu feedback nuevamente para saber si mejoro

**Comment by CarlNauj** [comment]
Buenas noches:

                Vea los videos y practique, no hay peor intento que el que no se hace, yo igual tengo mis dudas, pero voy a dar lo mejor, si fracaso, por lo menos lo intenté.

              Saluda atentamente a usted.

JUAN CARLOS CARREÑO POBLETE.

De: gladys5 ***@***.***>
Enviado el: martes, 1 de noviembre de 2022 17:18
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-frontend] Hola a todos,recien estoy empezando mis tareas muuuy atrazadas (#38)


Hola, Gracias por responder ,la verdad que si me veo todavia muy lejos de los objetivos del curso y espero resivir tu feedback nuevamente para saber si mejoro

-

**Comment by aldairxd04-garcia** [comment]
con que aplicacion hiciste tu buyyer persona? jsjs

**Comment by Hernanjas400** [comment]
Hola como estas prueba con Notion

**Comment by Rarriaza-sv** [comment]
https://github.com/orgs/Launch-X-Latam/teams/mision-frontend/discussions/40

**Comment by gladys5** [comment]
> con que aplicacion hiciste tu buyyer persona? jsjs

https://www.hubspot.es/make-my-persona?utm_medium=social&utm_source=youtube&utm_campaign=6ELCCOO4FEo_Espanol_youtube
saludos
